bionic/cache.py

```diff
@@ -862,10 +893,10 @@ class Provenance(object):
 
     @classmethod
     def from_computation(
-            cls, code_fingerprint, case_key, dep_provenances_by_task_key,
+            cls, code_fingerprint, case_key, dep_provenance_digest_by_task_key,
```

Comment:
Super nit: the convention here is to make the values plural, like `dep_provenance_digests_by_task_key`.

Reply:
I'll change it if it's the convention but I find it a little misleading, gives me the impression that there are multiple values for a task key.

Reply:
That's a fair response. Let's chat more about the convention. For now let's stick to it and consider revising things all at once in another PR?